Backend/webblog/blog/models.py

```diff
@@ -6,3 +6,10 @@ class Blog(models.Model):
     description = models.TextField(max_length=1500)
     created = models.DateTimeField(auto_now_add=True)
     owner = models.ForeignKey('auth.User', related_name='blog', on_delete=models.CASCADE, null=True)
+
+
+class Comment(models.Model):
+    blog = models.ForeignKey(Blog, related_name='comment', on_delete=models.CASCADE)
+    description = models.TextField(max_length=1500)
+    created = models.DateTimeField(auto_now_add=True)
+    owner = models.ForeignKey('auth.User', related_name='comment', on_delete=models.CASCADE, null=True)
```

Comment:
we should consider how we will handle threaded response.
<img width="505" alt="Screen Shot 2019-09-24 at 11 12 40 AM" src="https://user-images.githubusercontent.com/5072991/65487313-0de96f00-dec0-11e9-92b1-59bea202925d.png">

Reply:
it has been implemented now.